clearml_agent/helper/package/poetry_api.py

```diff
@@ -88,6 +88,11 @@ def _config(self, *args, **kwargs):
     @_guard_enabled
     def initialize(self, cwd=None):
         if not self._initialized:
+            # use correct python version -- detected in Worker.install_virtualenv() and written to
+            # session
+            if self.session.config.get("agent.python_binary", None) is not None:
```

Comment:
What if the setting is `agent.python_binary: ""`? Do we still want to take it?
Perhaps do:
```python
if self.session.config.get("agent.python_binary", None):
  ...
```

Reply:
Changed it accordingly.